src/spec/settings_spec.js

```diff
@@ -132,4 +132,29 @@ describe('Settings Page', () => {
     expect(settings.getDefaultDomain()).not.toEqual(defaultDomain);
     expect(settings.getDefaultDomain()).toEqual("fromdoppler.com");
   });
+
+  it('should set as enabled domain correctly', () => {

```

Comment:
In that case, there is something that I am not understanding, please call me when you can.